numtrip/game.py

```diff
@@ -0,0 +1,24 @@
+from random import randint
+from colorama import init
+from math import*
+MAX_POTENZ = 6
+spiel = [[(2**(randint(1, MAX_POTENZ+1))) for i in range(5)] for i in range(5)]
+anzeige = [[],[],[],[],[]] 
+print(spiel)
+for i in range(5):
+    for j in range(5):
+        if spiel[i][j] < 10:
+            anzeige[i].append(f"  {spiel[i][j]} ")
+        elif spiel[i][j] < 100:
+            anzeige[i].append(f" {spiel[i][j]} ")
+        elif spiel[i][j] < 1000:
+            anzeige[i].append(f"{spiel[i][j]} ")
+        print(spiel[i][j])
+print('╔══════╤══════╤══════╤══════╤══════╗')
+for i in range(5):
+    print("║      │      │      │      │      ║")
+    print("║",anzeige[i][0],"│", anzeige[i][1],"│",anzeige[i][2],"│",anzeige[i][3],"│",anzeige[i][4],"║")
```

Comment:
Zu Beachten: `print("||", "   4")` fügt zwoschen den beiden übergebenen Argumenten selber auch nochmals einen Leerschlag ein. Hier wollen Sie die Leerschläge aktiv kontrollieren. Daher besser, wenn Sie einen formatierten Text verwenden...
```suggestion
    print(f"║{anzeige[i][0]}│{anzeige[i][1]} usw...│",anzeige[i][2],"│",anzeige[i][3],"│",anzeige[i][4],"║")
```